cmd/ConfigLoader/main.go

```diff
@@ -52,6 +52,7 @@ func start(c *cli.Context) error {
 	ram := c.Int("ram")
 	hoursToReserve := c.Int("hourstoreserve")
 	osVersion := c.String("nodeosversion")
+	jenkinsjoburl := c.String("jenkinsjoburl")
```

Comment:
Since the UI for making a reservation may be something other than Jenkins in the future (although this is unlikely for us because of how tightly and seamlessly Jenkins integrates with the installer automation jobs), we should name this something more generic like "requestjoburl".  This way, if someone is using a different CI tool like Travis, CircleCI, or a custom UI, then the variable wouldn't be misleading

Reply:
sounds good.